tests/test_features_func_wrapper.py

```diff
@@ -85,6 +85,21 @@ def max_diff(x: pd.Series, mult=1):
     assert func2(dummy_data["EDA"]) == 0.25*3
 
 
-def test_error_func_wrapper_wrong_outputnames_type(dummy_data):
+def test_vectorized_func_wrapper(dummy_data):
```

Comment:
TODO:
- [ ] test for vectorized function on multiple series? 
- [x] test for vectorized function on irregularly sampled data (where it will just run I guess)